File: Basic/Functions/219158H.go
package main

import (
	"fmt"
)

func ceil(a, b int) int {
	add := 0
	if a % b != 0 {
		add++
	}
	return (a / b) + add
}

func floor(a, b int) int {
	return a / b
}

func round(a, b int) int {
	fl := float32(floor(a, b))
	diff := float32(a) / float32(b) - fl
	add := 0
	if diff >= float32(0.5) {
		add++
	}
	return int(fl) + add
}

func solution() {

	var a, b int
	fmt.Scan(&a, &b)

	fmt.Println("floor", a, "/", b, "=", floor(a, b))
	fmt.Println("ceil", a, "/", b, "=", ceil(a, b))
	fmt.Println("round", a, "/", b, "=", round(a, b))

}

func main() {
	TEST_CASE := !true
	t := 1
	if TEST_CASE {
		fmt.Scanf("%d", &t)
	}

	for t != 0 {
		solution()
		t--
	}
}

File: Basic/Functions/219158H.cpp
#include <iostream>
#include <cmath>
using namespace std;

#define intll long long
#define endl '\n'

void solution() {

	int a, b;
	cin >> a >> b;

	cout << "floor " << a << " / " << b << " = " << floor((double) a / b) << endl;
	cout << "ceil " << a << " / " << b << " = " << ceil((double) a / b) << endl;
	cout << "round " << a << " / " << b << " = " << round((double) a / b) << endl;
}

int32_t main() {

	// Test Case
	bool TEST_CASE = !true;

	ios::sync_with_stdio(false) ; cin.tie(0) ;

	int t_c = 1, tt_c = 1;
	if (TEST_CASE) cin >> t_c;

	// cin.ignore(); // In Case getline(cin, s)

	while (t_c--) {
		// cout << "Case " << tt_c++ << ": ";
		solution();
		// if (t_c) cout << '\n';
	}

	return 0;
}